app/src/main/kotlin/org/kotlinlsp/lsp/test.kt

```diff
@@ -0,0 +1,30 @@
+package app.ultradev.divineprison.util
```

Comment:
We can move this file to `testprojects/Basic/Autocomplete.kt`

Reply:
This one wasn't meant to be included, it was for me to analyze psi structure in intellij